src/mapblock_mesh.cpp

```diff
@@ -493,16 +494,14 @@ static void getNodeTextureCoords(v3f base, const v3f &scale, v3s16 dir, float *u
 
 struct FastFace
 {
-	TileLayer layer;
+	TileSpec tile;
```

Comment:
The decision of storing `layer` and not `tile` here was very strange. MeshCollector supports layers by itself.